pkg/lib/probe/probe.go

```diff
@@ -62,13 +77,72 @@ func NewDefaultProbe(target string, logger *zap.SugaredLogger) *Probe {
 					Host: targetURL.Hostname(),
 				},
 			},
-			TimeoutSeconds: _defaultTimeoutSeconds,
+			InitialDelaySeconds: _initialDelaySeconds,
+			TimeoutSeconds:      _defaultTimeoutSeconds,
+			PeriodSeconds:       _defaultPeriodSeconds,
+			SuccessThreshold:    _defaultSuccessThreshold,
+			FailureThreshold:    _defaultFailureThreshold,
 		},
 		logger: logger,
 	}
 }
 
-func (p *Probe) ProbeContainer() bool {
```

Comment:
this needs to be brought back and be called by `StartProbing`, because `IsHealthy` is currently not thread safe (will have race conditions, which can't happen)